app/controllers/GoldenTicketController.scala

```diff
@@ -188,8 +205,12 @@ object GoldenTicketController extends SecureCFPController {
       scala.concurrent.Future {
         Proposal.findById(proposalId) match {
           case Some(proposal) =>
+            val proposalIdsWithDelayedReview = ReviewByGoldenTicket.delayedReviewsReasons(uuid).keySet
+            val currentProposalReviewHasBeenDelayed = proposalIdsWithDelayedReview.contains(proposal.id)
```

Comment:
You can use Redis.hexists or Redis.exists also

Reply:
In the latest version of `GoldenTicketController.showVotesForProposal()` impl, I need to retrieve total number of delayed reviews (to display it)
=> introducing `allProposalIdsHavingDelayedReviewsForUser` through 42055757 should be enough for this purpose

Reply:
ok